index.html

```diff
@@ -0,0 +1,126 @@
+        <div class="flexcolum">
+          <img id="close" src="Img/CloseIcon.png" alt="Close" />
+          <ul class="ul flexcolum">
+            <li><a href="pages/about.html">About</a></li>
+            <li>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="heading">
+        <h3 class="margintop fontsec">"Hello frikies arround the world!"</h3>
+        <h1 class="margintop fontitul">Friki Corner <br />2023</h1>
+        <p class="description">
+          You welcome to the international convention about the leates releases in the comic world!
+          Come and meet people with the same likes and also enjoy the activities that we had
+          prepared for you. This amazing event will take place in December, in Mexico.
+        </p>
+        <h2>2023.12.17(sat) ~ 18(sun)</h2>
+        <p class="p">@Pepsi Center WTC, Mexico City</p>
+      </section>
+      <section class="program flexcolum">
+        <h3 class="subtit">Main Program</h3>
+        <div class="linea"></div>
+        <div class="mt">
+          <ul class="flexcolum ul flexrowDesk ul2">
+            <li class="flexrow flexcolumDesk">
+              <img src="Img/Program/contest.png" alt="Contest" />
+              <h4>Cosplay contest</h4>
+              <p>
+                Enroll or enjoy of the best cosplays in the contest where we have amazing prizes.
+              </p>
+            </li>
+            <li class="flexrow flexcolumDesk">
+              <img src="Img/Program/autographs.png" alt="autographs" />
+              <h4>Meet & Greet</h4>
+              <p>Meet your favorite voice actor or cartoonist and get an autograph and photo.</p>
+            </li>
+            <li class="flexrow flexcolumDesk">
+              <img src="Img/Program/shoping.png" alt="Shoping" />
+              <h4>Shopping</h4>
+              <p>Get some collectibles and limit edition from your favorite character.</p>
+            </li>
+            <li class="flexrow flexcolumDesk">
+              <img src="Img/Program/panels.png" alt="Panels" />
+              <h4>Panels and conferences</h4>
+              <p>Listen the directors and actors in our panels and conferences.</p>
+            </li>
+            <li class="flexrow flexcolumDesk">
+              <img src="Img/Program/gaming.png" alt="Gaming" />
+              <h4>Gaming</h4>
+              <p>Test the new game demo, and be able to participate in multilayer games.</p>
+            </li>
+          </ul>
+        </div>
+        <button class="mobile">Join to Friki Corner 2023</button>
+        <a href="#" class="desktop flexrowDesk">SEE THE WHOLE PROGRAM</a>
+      </section>
+      <section id="speakers" class="flexcolum">
+        <div class="flexcolum sp2">
+          <h3 class="subtit st2">Featured Speakers</h3>
+          <div class="linea mb"></div>
+        </div>
+      </section>
```

Comment:
great work done by making this section generate content dynamically , one very small detail is the width that this section takes and the text of each item by reducing the width the text will wrap as well 

![image (6)](https://github.com/AndreaM2429/Capstone_project_Conference_page/assets/59707859/c3ba16f9-63e3-4e45-997c-a99022e68416)
![image (7)](https://github.com/AndreaM2429/Capstone_project_Conference_page/assets/59707859/bddabf8b-1bf2-407c-8c56-e9b9201e5ff3)

Reply:
Done